source/examples/rapids-sagemaker-higgs/rapids-higgs.py

```diff
@@ -4,8 +4,8 @@
 
 import cudf
 from cuml import RandomForestClassifier as cuRF
-from cuml.preprocessing.model_selection import train_test_split
-from sklearn.metrics import accuracy_score
+from cuml.metrics import accuracy_score
+from cuml.model_selection import train_test_split
```

Comment:
There is no `cuml.preprocessing.model_selection`.

`train_test_split` comes from `cuml.preprocessing`.

This fixes this error:

```text
Traceback (most recent call last):
  File "/opt/ml/code/rapids-higgs.py", line 7, in <module>
    from cuml.preprocessing.model_selection import train_test_split
ModuleNotFoundError: No module named 'cuml.preprocessing.model_selection'
```